website/docs/r/eks_node_group.html.markdown

```diff
@@ -35,6 +35,28 @@ resource "aws_eks_node_group" "example" {
 }
 ```
 
+### Ignoring Changes to Desired Size
+
+You can utilize the generic Terraform resource [lifecycle configuration block](/docs/configuration/resources.html#lifecycle) with `ignore_changes` to create an EKS Node Group with an initial size of running instances, then ignore any changes to that count caused externally (e.g. Application Autoscaling).
```

Comment:
will the docs file be added to this repo? wasn't sure if there's an upcoming change there or if we want to refer to the external lifcycle link: https://www.terraform.io/docs/configuration/resources.html?&_ga=2.203417043.394886334.1589905661-628874299.1583248857#lifecycle-lifecycle-customizations

Reply:
The general terraform.io/docs files live in https://github.com/hashicorp/terraform-website and providers will not be expected to maintain them separately at any point. When the provider documentation is officially migrated to the Terraform Registry, then we will change all these absolute path links with fully absolute links (e.g. https://terraform.io/docs/...), but this is waiting to be done all at once as far as I know. I will update the anchor so its more accurate though.

Reply:
(For more context: absolute path links to /docs/... in the Registry documentation are automatically rewritten to point at https://terraform.io/docs/... to not make this a problem during the transition period -- not sure if this is done during serving the documentation or during provider ingress off the top of my head)

Reply:
ah no wonder, cool make sense!